proc-macros/src/lib.rs

```diff
@@ -179,7 +179,9 @@ pub(crate) mod visitor;
 /// **Arguments:**
 ///
 /// - `name` (mandatory): name of the RPC method. Does not have to be the same as the Rust method name.
-/// - `unsub` (mandatory): name of the RPC method to unsubscribe from the subscription. Must not be the same as `name`.
+/// - `unsubscribe` (optional): name of the RPC method to unsubscribe from the subscription. Must not be the same as `name`.
+/// - `aliases` (optional): aliases for `name`.
```

Comment:
We should mention how this interacts with the namespace setting as well (or rather: how it **doesn't** interact with it).